fili-core/src/main/java/com/yahoo/bard/webservice/jobs/ApiJobStoreFilterOperation.java

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE file distributed with this work for terms.
+package com.yahoo.bard.webservice.jobs;
+
+/**
+ * Types of legal ApiJobStoreFilter operations.
+ */
+public enum ApiJobStoreFilterOperation {
+    eq
```

Comment:
That's right. I am working on some final changes for #23 after which I will rebase